genny/auth/templates/models/user.go.plush

```diff
@@ -31,15 +35,48 @@ type User struct {
 // Create wraps up the pattern of encrypting the password and
 // running validations. Useful when writing tests.
 func (u *User) Create(tx *pop.Connection) (*validate.Errors, error) {
-	u.Email = strings.ToLower(u.Email)
-	ph, err := bcrypt.GenerateFromPassword([]byte(u.Password), bcrypt.DefaultCost)
+	verrs := u.validatePassword()
+	if verrs.HasAny() {
+		return verrs, nil
+	}
+
+	ph, err := encryptPassword(u.Password)
 	if err != nil {
 		return validate.NewErrors(), errors.WithStack(err)
 	}
 	u.PasswordHash = string(ph)
+	u = u.sanitizeFields()
 	return tx.ValidateAndCreate(u)
 }
 
+// Update handles the extra work possibly needed during user update,
+// hashing password and making email lowercase for consistency
```

Comment:
In general, the documentation for `Create` and `Update` are quite different, maybe we can make them more similar?

Reply:
Please let me know if this is still to different.